deploy/chart/README.md

```diff
@@ -57,10 +57,8 @@ The following table lists the configurable parameters of the cert-manager chart
 | `image.tag` | Image tag | `canary` |
```

Comment:
Also update this readme with the `00-crds.yaml` install step, or remove all the install steps and just point at the docs